drivers/sensor/tmd26xx/tmd26xx.h

```diff
@@ -0,0 +1,322 @@
+/*
+ * SPDX-FileCopyrightText: Copyright (c) 2023 PHYTEC Messtechnik GmbH
+ * SPDX-FileCopyrightText: Copyright (c) 2023 Carl Zeiss Meditec AG
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr/drivers/gpio.h>
+
+#ifndef ZEPHYR_DRIVERS_SENSOR_TMD26XX_TMD26XX_H_
+#define ZEPHYR_DRIVERS_SENSOR_TMD26XX_TMD26XX_H_
+
+#define TMD2635_CHIP_ID 0X44
```

Comment:
If possible please move the register defines to tmd26xx_reg.h. Access to these defines might come handy when developing an emul and eventually testing the driver